src/main/java/com/softteam/vocabuilder/controller/CategoryController.java

```diff
@@ -0,0 +1,47 @@
+package com.softteam.vocabuilder.controller;
+
+import com.softteam.vocabuilder.persistence.entity.Category;
+import com.softteam.vocabuilder.service.CategoryServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.persistence.MapsId;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/categories")
+
+public class CategoryController {
+
+    @Autowired
+    private CategoryServiceImpl categoryService;
+    @PostMapping
```

Comment:
could you let some empty lines? between the attributes and methods to improve the readability